### PR TITLE
Docs: Add note about barycentric coordinates in triangle.containsPoint() docs

### PR DESCRIPTION
**Pull request by donmccurdy**
Open to better ways of explaining this.. I assumed incorrectly that `triangle.containsPoint(v)` would always return false for points distant from the plane of the triangle.

### CODE REVIEW

**Review by Mugen87 — APPROVED** [review]

**Comment by looeee** [comment]
I'm not sure that barycentric coordinates and projecting the point onto the triangle before projecting are related. 

You could just say that the point is projected onto the triangle before checking.

**Comment by WestLangley** [comment]
I think we should fix the code.

>I assumed incorrectly that triangle.containsPoint(v) would always return false for points distant from the plane of the triangle.

The source code of `triangle.closestPointToPoint()` appears to assume the same.

/ping @bhouston Any opinion on the best solution here?

**Comment by bhouston** [comment]
The issue is that in 3D floating point math you are never truly on a plane, you are always a fraction off.  I didn't want to introduce an eplison (which are always wrong for some cases), thus it is really are you in the infinite prism extending in front and behind the triangle.

Maybe we could introduce another parameter that is maximumDistanceFromTrianglePlane?  This way it is at least explicit.  Maybe it defaults to infinity to be backwards compatible?

File: docs/api/math/Triangle.html [hunk 1]
```diff
@@ -79,7 +79,9 @@ <h3>[method:Boolean containsPoint]( [page:Vector3 point] )</h3>
 		<div>
 		[page:Vector3 point] - [page:Vector3] to check.<br /><br />
 
-		Returns true if the passed vector is within the triangle.
+		Returns true if the passed vector is within the triangle in
+		[link:https://en.wikipedia.org/wiki/Barycentric_coordinate_system barycentric coordinates]
+		(the vector is projected onto the plane of the triangle before checking containment).
```
Comment (WestLangley) [inline docs/api/math/Triangle.html]: Maybe
```
Returns true if the passed point, when projected onto the plane of the triangle, lies within the triangle.
```

Comment (donmccurdy) [reply docs/api/math/Triangle.html]: Updated, with small rearrangement:

```
Returns true if the passed point lies within the triangle, when projected onto the plane of the triangle.
```

Comment (donmccurdy) [reply docs/api/math/Triangle.html]: Never mind, going with your suggested wording.

**Comment by donmccurdy** [comment]
Updated the wording to omit implementation details about barycentric coordinates.

File: docs/api/math/Triangle.html (hunk 1 above)
Comment (WestLangley) [reply docs/api/math/Triangle.html]: Hehe. I had trouble wording that one, too. :)

**Comment by WestLangley** [comment]
Given the change to the wording of the docs, I'm happy leaving the code as-is.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**